Makefile.core.mk

```diff
@@ -14,6 +14,9 @@
 
 SHELL := /bin/bash -o pipefail
 
+FINDFILES=find . \( -path ./common-protos -o -path ./.git -o -path ./out -o -path ./.github -o -path ./licenses -o -path ./vendor -o -path ./cmd/gen-release-notes/templates \) -prune -o -type f
```

Comment:
I made a comment in https://github.com/istio/common-files/pull/960#issuecomment-1852350517 where we could simply add an additional variable here, like FINDFILES_IGNORE and then add that to the find command in common-files.

Reply:
Makes sense, thanks